docs/index.md

```diff
@@ -10,11 +10,24 @@ description: |-
 
 The provider can be used to interact with [Juju][0] - a model-driven Operator Lifecycle Manager (OLM) by Canonical.
```

Comment:
According to the current Juju terminology guidelines, we should drop the "OLM" verbiage.

Reply:
What's the replacement? Going with `an open source orchestration engine by Canonical.` from juju.is for now.